src/vuln_analysis/utils/java_utils.py

```diff
@@ -2013,6 +1945,8 @@ def vid_of(rec: Dict) -> Tuple[str, str]:
 
     def pick_variant(fq: str, jar: str) -> Dict | None:
         """Pick the variant in the same jar if present; else unique globally; else None."""
+        if not fq:
```

Comment:
@tmihalac I Guess you meant fully qualified? if so, please change it so it will be more clear.
if not, change the name to the purpose of this parameter/argument so it will reflect what it does.